### ISSUE TITLE
mesh face and edge coordinate variables

### ISSUE THREAD

**Issue by marqh** [ugrid-conventions/ugrid-conventions#34]
Hello Ugridders

I would like to investigate the value of the 
```
mesh face and edge coordinate variables
```

It appears to me that this is an over-specification, particularly should #33 be adopted.

An edge is defined by it's nodes, so given an edge definition and its defining nodes, I know where the edge is all the way along, so to speak.

What is the key use case for providing a 'coordinate' who's job is to tell me where an edge centre is?

What should an application do if the edge centre appears not to be on the edge, given the nodes which define it? 

thank you
mark

**Comment by ChrisBarker-NOAA** [comment]
You are right, those coordinate variables are all over-specified. Which is why they are optional.

But they are there both to provide greater CF compliance and more convenient access for common use cases like plotting, etc.

Without them, you would have to find them by computing them through a couple levels of indirection:

figure out which nodes define them
get the coordinates of those nodes
compute the center of the [edge, face, cell] from those coordinates.

Not sure why #33 makes a difference here ?

Though it's a reminder that we should probably specify that everything should be given in the same coordinate system.
